example/patches/react-native-windows+0.67.3.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/node_modules/react-native-windows/Mso/dispatchQueue/dispatchQueue.h b/node_modules/react-native-windows/Mso/dispatchQueue/dispatchQueue.h
+index 0a7369e..e909b93 100644
+--- a/node_modules/react-native-windows/Mso/dispatchQueue/dispatchQueue.h
++++ b/node_modules/react-native-windows/Mso/dispatchQueue/dispatchQueue.h
```

Comment:
This is a merged PR upstream but not released yet. patch-package will nicely give us a warning in the future while still working